content/4_Gold/Ternary_Search.mdx

```diff
@@ -0,0 +1,151 @@
+---
+id: ternary-search
+title: Ternary Search
+author: Chongtian Ma
+prerequisites:
+  - /silver/binary-search
+redirects:
+  - /bronze/ternary-search
+  - /gold/ternary-search
+  - /plat/ternary-search
+description: Searching for a peak in a unimodal function.
+frequency: 1
+---
+
+<Resources>
+	<Resource
+		source="CP-Algorithms"
+		title="Ternary Search"
+		url="https://cp-algorithms.com/num_methods/ternary_search.html"
+		starred
+	/>
+</Resources>
+
+Ternary search is the equivalent of binary searching for the zero on the
+derivative function. The most basic function can be represented by the standard
```

Comment:
should we assume knowledge of calculus

Reply:
idk, if they dont udnerstand it its not big of a deal

Reply:
if it's not that big of a deal then remove it